docker/data-science/notebooks/ch05/1.open-information-extraction.ipynb

```diff
@@ -10,26 +10,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
+    "\n",
     "import sys\n",
     "sys.path.append('..')\n",
     "from aips import *\n",
+    "import spacy\n",
```

Comment:
this looks redundant

Reply:
I'm going to go through all the notebooks at some point and execute python's isort